version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.30-SNAPSHOT"
```

Comment:
It looks like the version numbers weren't changed after the last release, so I'm doing it manually here. Is this the best way to fix things @rupertbates?